src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment, key }) => (
+  <>
+    <div key={key} className="CommentInfo">
```

Comment:
```suggestion
    <div className="CommentInfo">
```
React fragment is redundant here as you already have a container and you render only one element.
Also, there is no need to accept key attribute. Just add it in the parent component on mapping
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/96409563/626b1ea3-9b14-4d45-8cc4-f5de42277f95)